src/components/buttons/accessible-button/accessible-button.spec.js

```diff
@@ -31,14 +31,17 @@ describe('rendering', () => {
     expect(container).toContainElement(ref.current);
   });
   it('should be marked as "disabled"', () => {
+    const onClick = jest.fn();
     const { getByLabelText } = render(
-      <AccessibleButton {...props} isDisabled={true} />
-    );
-    expect(getByLabelText('test-button')).toHaveAttribute('disabled');
-    expect(getByLabelText('test-button')).toHaveAttribute(
-      'aria-disabled',
-      'true'
+      <AccessibleButton {...props} onClick={onClick} isDisabled={true} />
     );
+    const button = getByLabelText('test-button');
+
+    expect(button).toHaveAttribute('disabled');
+    expect(button).toHaveAttribute('aria-disabled', 'true');
+
+    button.click();
+    expect(onClick).not.toHaveBeenCalled();
```

Comment:
before we couldn't test this as `pointer-events: none` doesn't stop you from calling onClick manually 😆

Reply:
although TBH testing `disabled` is kind of dumb